students/alex_skrn/lesson02/grid.py

```diff
@@ -0,0 +1,112 @@
+"""Print a 2x2 grid, define two print_grid functions."""
+# PART 1 - print a 2x2 grid, each cell 4 dashes large
+
+plus, minus, bar, space = '+', '-', '|', ' '
+numcells = 2
+numdashes = 4
+
+# construct the plus-and-dashes line, + - - - - + - - - - +
+plusline = ''
+for i in range(numcells):
```

Comment:
When you aren't using the variable in the for loop, like `i` here, common usage is to use an `_` so other programmers know that the variable is a throw away.
```
for _ in range(num_cells):
```